scripts/west_commands/sign.py

```diff
@@ -109,6 +109,8 @@ def do_add_parser(self, parser_adder):
                            are currently supported''')
         group.add_argument('-p', '--tool-path', default=None,
                            help='''path to the tool itself, if needed''')
+        group.add_argument('-s', '--sign-schema', required=False, default=None,
+                           help='''signing schema used with rimage''')
```

Comment:
One typo in commit: "sign schama" -> "sign schema" 

Also the rationale is not entirely correctly. Zephyr does allow to specify different schema per target, but we don't have TGL-H support in Zephyr. I'm adding this in https://github.com/zephyrproject-rtos/zephyr/pull/41764

Maybe better wording: 
```
For an intel ADSP platform, the rimage sign schema
is not fixed. For example intel_adsp_cavs25 platforms can
support two IPC variants (IPC3 and IPC4) and the two 
require difference signing scheme to be used. 

This patch adds an option, so user can specified
rimage sign schema when use 'west sign'.
```